packages/graphhopper/index.ts

```diff
@@ -278,6 +294,14 @@ export class GraphHopper implements BaseRouter {
                     return res
                 }
             })
+            .catch((error)=>{
```

Comment:
We can remove redundancy by defining the catch fallback function once and pass it in each method (it's the same after all across endpoints)